src/main/java/com/globant/carrito/StatusDto.java

```diff
@@ -1,13 +1,13 @@
 package com.globant.carrito;
 
 public class StatusDto {
-	private boolean ok;
+	private boolean status;
```

Comment:
jeje si dale, lo cambio :^p

Date: Wed, 4 Feb 2015 07:02:50 -0800
From: notifications@github.com
To: cart_client@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [cart_client] Avance del shopping cart hasta el 03/02/15. (#1)

In src/main/java/com/globant/carrito/StatusDto.java:

> @@ -1,13 +1,13 @@
>  package com.globant.carrito;
>  
>  public class StatusDto {
> -	private boolean ok;
> +	private boolean status;

Fijate de cambiar el código javascritp para que valide contra status en lugar de hacerlo contra ok. status es un mejor nombre, jeje.


—
Reply to this email directly or view it on GitHub.